cg/cli/add.py

```diff
@@ -62,6 +62,16 @@ def add():
     show_default=True,
     required=False,
 )
+@click.option(
+    "-ic",
+    "--is-clinical",
+    "is_clinical",
+    help="Flag stating if it is a clinical customer.",
```

Comment:
Maybe for the users of this function may be obvious, but a short explanation of what is a clinical customer would be better, as the flag is self-explanatory about the status but not about the meaning.

Reply:
Reasonable, will add that this will be used to determine for how long the data will be stored.